docs/0.3.5-alpha/stdlib/doc/array.md

```diff
@@ -1,26 +1,57 @@
 ## `array_first_index`
+
+```ab
+import { array_first_index } from "std/array.ab"
```

Comment:
`import { array_first_index } from "std/array.ab"` does not work for me. I will replace this also.

Reply:
Done by `sed -i -E 's/^(import .+ from "std\/.+)\.ab"$/\1"/g' ../amber-docs/docs/0.3.5-alpha/stdlib/doc/*`